src/app/assess3/layout/assess3-layout.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit, ViewEncapsulation, AfterViewInit, ChangeDetectorRef } from '@angular/core';
+import { Location } from '@angular/common';
+
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { Observable } from 'rxjs/Observable';
+import { Store } from '@ngrx/store';
+
+import * as assessActions from '../store/assess.actions';
+import * as assessReducers from '../store/assess.reducers';
+
+import { AssessStateService } from '../assess-state.service';
+
+@Component({
+  selector: 'assess3-layout',
+  templateUrl: './assess3-layout.component.html',
+  styleUrls: ['./assess3-layout.component.scss'],
+})
+export class Assess3LayoutComponent implements OnInit, AfterViewInit {
+
+  public title = new BehaviorSubject('').asObservable();
+  public showBackButton = new BehaviorSubject(false).asObservable();
+
+  public constructor(
+    private store: Store<assessReducers.AssessState>,
+    private location: Location,
+    private changeDetectorRef: ChangeDetectorRef,
+  ) { }
+
+  /**
+   * @description
+   * @returns {void}
+   */
+  public ngOnInit(): void {
+    console.log('&&&&&&&&&& Got to assess3 Assess3LayoutComponent!');
```

Comment:
I know it's just an early prototype, but this style of log should probably not go into the baseline.